Packs/Sixgill-Darkfeed/Integrations/Sixgill_DVE_Feed/Sixgill_DVE_Feed.yml

```diff
@@ -0,0 +1,120 @@
+category: Data Enrichment & Threat Intelligence
+commonfields:
+  id: Sixgill_DVE_Feed
+  version: -1
+configuration:
+- display: Client Id
+  hidden: false
+  name: client_id
+  required: true
+  type: 0
+- display: Client Secret
+  hidden: false
+  name: client_secret
+  required: true
+  type: 4
+- defaultvalue: '1000'
+  hidden: false
+  name: maxIndicators
+  required: false
+  type: 0
```

Comment:
Please add a display here like you did in your previous feed: 
![image](https://user-images.githubusercontent.com/50294648/112889279-1a1a4200-90de-11eb-9ef3-1a5262df8da2.png)